### PR TITLE
Using IoT Core as scalable ingest for hybrid projects

### PR DESCRIPTION
**Pull request by ptone**

### CODE REVIEW

File: tutorials/cloud-iot-hybrid/index.md [hunk 1]
```diff
@@ -0,0 +1,248 @@
...
+date_published: 2018-05-30
+---
+
+Preston Holmes | Solution Architect | Google
+
+This tutorial demonstrates how to use [Cloud IoT Core](https://cloud.google.com/iot) and [Cloud PubSub](https://cloud.google.com/pubsub/) to provide a secure and scalable ingest layer, combined with a small relay service over private networking to an on-prem IoT solution.
+
+## Objectives
+
+- Configure IoT Core and Pubsub as a fully managed ingest system
```
Comment (dghale) [inline tutorials/cloud-iot-hybrid/index.md]: %s/pubsub\/c/Cloud Pub\/Sub/

File: tutorials/cloud-iot-hybrid/index.md [hunk 2]
```diff
@@ -0,0 +1,248 @@
...
+
+## Verifying Traffic Flow
+
+You can now use an MQTT client to connect to the stand-in for the on-prem broker. This would represent some part of the on-prem IoT application. One simple browser based tool you can use in Chrome is [MQTTLens](https://chrome.google.com/webstore/detail/mqttlens/hemojaaeigabkbcookmlgmdigohjobjm?hl=en).
+
+Use the public IP address of the `on-prem-rabbit` instance, along with a username of `user` and password of `abc123` to connect.
+
+Create a subscription to an MQTT topic of `sample`.
+
+Now you can use one of the [IoT Core quickstart samples](https://cloud.google.com/iot/docs/quickstart) to send test messages to the topic of
```
Comment (dghale) [inline tutorials/cloud-iot-hybrid/index.md]: ...to the topic of:

File: tutorials/cloud-iot-hybrid/index.md [hunk 3]
```diff
@@ -0,0 +1,248 @@
...
+
+Use the public IP address of the `on-prem-rabbit` instance, along with a username of `user` and password of `abc123` to connect.
+
+Create a subscription to an MQTT topic of `sample`.
+
+Now you can use one of the [IoT Core quickstart samples](https://cloud.google.com/iot/docs/quickstart) to send test messages to the topic of 
+
+`/devices/${deviceId}/events/sample`
+
+You can also quickly emulate this by publishing directly to the pubsub topic directly with a `subFolder` attribute.
```
Comment (dghale) [inline tutorials/cloud-iot-hybrid/index.md]: Cloud Pub/Sub

File: tutorials/cloud-iot-hybrid/index.md [hunk 4]
```diff
@@ -0,0 +1,248 @@
...
+```
+
+The relay republishes this to the corresponding MQTT topic on the on-prem broker over the private network, in this case using the project-level private network DNS to lookup `on-prem-rabbit` host.
+
+## Next Steps
+
+This relay service could be adapted in a number of ways:
+
+- scaled out to consume more messages in parallel from pubsub - perhaps deployed to [Kubernetes Engine](https://cloud.google.com/kubernetes-engine/)
+- use rate limiting to protect the on-prem broker from overload, using the pubsub subscription as a "surge tank"
```
Comment (dghale) [inline tutorials/cloud-iot-hybrid/index.md]: Capitalize list items and end each with a period.

File: tutorials/cloud-iot-hybrid/index.md [hunk 5]
```diff
@@ -0,0 +1,248 @@
...
+
+You might also consider having the relay pulling from Pubsub and writing more directly to alternate services on-prem, instead of through an on-prem MQTT broker. However by relaying MQTT - it lets the on-prem development proceed if MQTT was already built into the solution, or if a looser coupling is wanted.
```
Comment (dghale) [inline tutorials/cloud-iot-hybrid/index.md]: Cloud Pub/Sub

replace dash with a comma: "...relaying MQTT, it lets the..."

File: tutorials/cloud-iot-hybrid/index.md [hunk 6]
```diff
@@ -0,0 +1,248 @@
...
+## Create the "on-prem" broker
+
+You will use the project private networking as a stand-in for a proper hybrid [Google Cloud Interconnect](https://cloud.google.com/interconnect/) setup.
+
+
+Start by creating a VM to represent the on prem broker instance. This will be running a basic version of [RabbitMQ](https://www.rabbitmq.com/).
+
+```sh
+gcloud beta compute instances create-with-container on-prem-rabbit \
+--zone=us-central1-c \
```
Comment (dghale) [inline tutorials/cloud-iot-hybrid/index.md]: Could you use $CLOUD_ZONE here?

**Comment by ptone** [comment]
Thanks @gguuss and @dghale  - I've incorporated your excellent feedback as well as some offline feedback from @chut

I ran through it end-end after resetting my cloud-shell to remove any lurking undocumented assumptions.

**Comment by ptone** [comment]
@dghale I believe I addressed your comments, my understanding is to unblock the merge, I need an LGTM from you here on the PR - would love to get this merged next week

**Comment by dghale** [comment]
LGTM

**Comment by gguuss** [comment]
LGTM

**Pull request merged**